infra/configs.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The name of the GCP project
+# TODO ameukam: (use kubernetes-public instead)
+PROJECT="k8s-infra-dev-cluster-turnup"
+
+# The name of the GKE cluster
+# TODO wg-k8s-infra: define nomenclature for the name
+NAME="${NAME:-gke-cluster}"
+
+# The IAM service acccount for GKE
+SVC_ACCOUNT="gke-node-sa"
+
+# The BigQuery dataset name for billing data (don't support special characters  such as -, &, @, or %)
+DATASET_NAME="k8s_cluster_test_billing"
+
+# The name of the GCP region where the cluster will be created
+REGION="${REGION:-us-central1}"
+
+# The Version the GKE cluster
+CLUSTER_VERSION="gcloud beta container get-server-config --project=${NAME} --region=${REGION} --format='value(validMasterVersions[0])'"
```

Comment:
This means we will always manually fetch whatever the first version that is returned, even if it's not the recommended default.

Reply:
Sorry, forgot to remove it!